CalorieTracker/CalorieTracker/Model/CalorieTracker.xcdatamodeld/CalorieTracker.xcdatamodel/contents

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="14492.1" systemVersion="18G87" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="CalorieCount" representedClassName="CalorieCount" syncable="YES" codeGenerationType="class">
+        <attribute name="date" optional="YES" attributeType="Date" usesScalarValueType="NO" syncable="YES"/>
```

Comment:
I think you should consider naming this `timestamp` given that timestamp is a slightly more appropriate term for what this CoreData property is being used for.